packages/frontend/src/index.js

```diff
@@ -16,7 +17,9 @@ root.render(
       <FormProvider>
         <PhotosProvider>
           <GameProvider>
-            <App />
+            <BreakpointProvider>
+              <App />
+            </BreakpointProvider>
```

Comment:
I think the breakpoint provider should have all the other providers as children, so we could use the breakpoints inside the other providers if needed at some point